.github/workflows/package.yml

```diff
@@ -7,29 +7,33 @@ on:
 
 jobs:
   publish:
-
     name: Publish
-
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v2
 
       - uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: 3.9.2
+
+      - name: Run Poetry image
+        uses: abatilo/actions-poetry@v2.0.0
+        with:
+          poetry-version: 1.1.4
 
       - name: Building
         run: >-
-          python setup.py sdist
+          make build
 
+      # TODO(dmu) LOW: Consider publishing with Poetry
```

Comment:
@buckyroberts Is there a way to test publishing to PyPI before actually merging to `master`? Can github read workflows definitions from any branch (like bitbucket does for gitflow) not just `master` or `development` branch? I could not quickly find a way to do it.

Reply:
We can do:

```
# Triggered when code is pushed to any branch in a repository
on: push
```

or...


```
# Triggers the workflow on push or pull request events
on: [push, pull_request]
```

Reply:
Thank you. It turned out to work even with new branches.